index.html

```diff
@@ -0,0 +1,24 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>UI Challenge</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+        <div class='a'>a</div>
+        <div class='b'>b</div>
+        <div class='c'>c</div>
+        <div class='d'>d</div> 
+        <div class='float'>
+            <div class='e'>e</div>
+            <div class='f'>f</div>
+            <div class='g'>g</div>
+            <div class='h'>h</div>
+            <div class='i'>i</div>
+        </div>
+        <!-- <div class='i'>i</div> -->
+
+    </body>
+</html>
```

Comment:
Tiny nitpick: This angry red symbol means that you didn't leave an extra blank line at the end of your file. GitHub (and most systems) recommend leaving one blank line at the end of the file, for code review purposes.